rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -241,15 +239,16 @@ A node MUST make sure to only add transactions of which all previous transaction
 
 The `State` message is sent as response to various conditions of the gossip protocol (see §5).
 The `State` message MUST contain a new `conversationID`.
-The `State` message contains a `XOR` value and an `LC`.
+The `State` message contains an `XOR` value and an `LC`.
 The `LC` value MUST equal the highest Lamport Clock value of all transaction references included in the `XOR` calculation.
 If no transactions are present, an all-zero `XOR` and `LC` of 0 is sent.
 
-This might look exactly like the `Gossip` message, but the type of response is different so it is a different type of message.
+This might look exactly like the `Gossip` message, but the type of response is different, so it is a different type of message.
```

Comment:
Gossip also has a `transactions` field, so they don't look exactly the same to me

Reply:
Useless remark for a spec, will remove